schema/information_schema_1-2-0.json

```diff
@@ -0,0 +1,241 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://Orange-OpenSource.github.io/its-client/schema/info",
+  "description": "Information JSon schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "version",
+    "instance_id",
+    "instance_type",
+    "running",
+    "timestamp",
+    "validity_duration"
+  ],
+  "properties": {
+    "type": {
+      "description": "type of server",
+      "type": "string",
+      "enum": [
+        "broker"
+      ]
+    },
+    "version": {
+      "type": "string",
+      "description": "json message format version",
+      "const": "1.1.99"
```

Comment:
1.2.0, a simple way to force the client to well share the one used

Reply:
The version is actually bumped *after* we do all the changes.

Note that this version, 1.1.99 is an exact copy of the 1.1.0 version, with just the version field bumped.